test/domain_bridge/test_domain_bridge_end_to_end.cpp

```diff
@@ -142,7 +146,6 @@ TEST_F(TestDomainBridgeEndToEnd, remap_topic_name)
   bridge.bridge_topic(
     topic_name, "test_msgs/msg/BasicTypes", kDomain1, kDomain2, topic_bridge_options);
 
-  EXPECT_TRUE(poll_condition([sub]() {return sub->get_publisher_count() == 1u;}, 3s));
```

Comment:
I guess waiting for the match is still good to do :shrug:

Reply:
We could either add a retry (if you think it will help) or skip these tests for rmw_connextdds.

Reply:
I did some extra testing locally, it seems to be that connext is just too slow and 3s is sometimes not enough.